CIP-Improved_Rewards_Scheme_Parameters/README.md

```diff
@@ -0,0 +1,239 @@
+
+Stakepool Competitive Point: 1 ADA
+
+| Pool Type | Pool Stake        | Pool Pledge      | Pool Epoch Revenue | Delegator ROS |
+|-----------|-------------------|------------------|--------------------|---------------|
+| Sybil     | 100,000.00 ADA    | 100,000.00 ADA   | 1.52 ADA           | 3.6615%       |
+| Sybil     | 1,000,000.00 ADA  | 1,000,000.00 ADA | 15.24 ADA          | 3.6620%       |
+| Community | 10,000,000.00 ADA | 100,000.00 ADA   | 152.49 ADA         | 3.6639%       |
+| Community | Saturated         | 1,000,000.00 ADA | 720.44 ADA         | 3.6853%       |
+
+#### Proposed - Stage 4
+
+Stakepool Viability Point: 1 ADA
+
+Stakepool Competitive Point: 1 ADA
+
+| Pool Type | Pool Stake        | Pool Pledge      | Pool Epoch Revenue | Delegator ROS |
+|-----------|-------------------|------------------|--------------------|---------------|
+| Sybil     | 100,000.00 ADA    | 100,000.00 ADA   | 1.52 ADA           | 3.6615%       |
+| Sybil     | 1,000,000.00 ADA  | 1,000,000.00 ADA | 15.24 ADA          | 3.6624%       |
+| Community | 10,000,000.00 ADA | 100,000.00 ADA   | 152.52 ADA         | 3.6646%       |
+| Community | Saturated         | 1,000,000.00 ADA | 542.82 ADA         | 3.6932%       |
+
+### Backward Compatibility
+
+This proposal includes several parameter changes and changes to ledger rules. Specifically, stage 2 of this proposal will require a hardfork to introduce a new parameter, delete a parameter, and modify the stakepool fee calculation equation. As mentioned in the specification section, the stakepool fee calculation equation must be modified in order to ensure current stakepool registration certificates are compatible with this CIP.
+
+## Path to Active
+
+### Acceptance Criteria
+
+#### For Stages 1, 3, and 4
+1. The raw transaction for the parameter update is built.
+2. Genesis delegates IOG, CF, and Emurgo sign the transaction.
+3. Transaction is submitted.
+4. Parameter update is accepted by majority of the network.
+5. Parameter update is confirmed.
+
+#### For Stage 2
+
+1. IOG completes necessary research and develops the changes to the ledger rules.
+2. IOG releases a new version of cardano-node supporting the changes to the ledger rules.
+2. The raw transaction signalling the hardfork is built.
+3. Genesis delegates IOG, CF, and Emurgo sign the transaction.
+4. Transaction is submitted.
+5. Hardfork is accepted by majority of the network.
+6. Hardfork and changes to ledger rules are confirmed.
+
+### Implementation Plan
+
+Implementors include IOG, CF, and Emurgo. All three entities must approve the changes to the network. Each stage will be an individual update. Stages 1, 3, and 4 will be parameter updates. Stage 2 will require a hardfork.
+
+Before implementation, IOG's engineering and research teams must review the feasibility and potential consequences of the proposal. IOG will create the implementation for each update and decide on the time interval between updates.
```

Comment:
```suggestion
Each stage will be an individual update. Stages 1, 3, and 4 will be parameter updates. Stage 2 will require a hardfork.

Before implementation, engineering and research teams must review the feasibility and potential consequences of the proposal, create the implementation for each update, and decide on the time interval between updates.
```
Like we said in an earlier comment (maybe for your other proposal... having to duplicate everything makes me uncertain), only companies who've agreed to participate in the implementation should be mentioned explicitly here.

It makes the language a bit awkward and we'll see if @KtorZ has any feedback that might relax the requirement to avoid mentioning these companies explicitly.  My feeling is that the `Inactive` status should allow authors to say anything they want here.